Formula/forgit.rb

```diff
@@ -0,0 +1,17 @@
+class Forgit < Formula
+  desc "Interactive git commands in the terminal"
+  homepage "https://github.com/wfxr/forgit"
+  url "https://github.com/wfxr/forgit/releases/download/23.04.0/forgit-23.04.0.tar.gz"
+  sha256 "89826d12b551829c4bce396551b1505c0604ba29130f4b8e2f842c2ca0b4c9e3"
+  license "MIT"
+
+  depends_on "fzf"
+
+  def install
+    bin.install "bin/git-forgit" => "git-forgit"
+  end
+
+  test do
+    system "which", "git-forgit"
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests.

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
Hi @chenrui333, thanks for your feedback. `forgit` is a fully interactive tool, so I actually don't know how to test any functionality of it without user input. Do you have an idea?

Reply:
To explain it even further: `forgit` is an interactive frontend for `git`. So the actual *functionality* that coud be tested is just functionality of `git`. Everything `forgit` does is just user presentation. How could we test this?

Reply:
Nevermind, I think I found a useful testcase. `git forgit add <file>` bypasses the interactive selector while still checking if forgit is correctly installed, because otherwise `git` would complain:

`git: 'forgit' is not a git command. See 'git --help'.`

So this should be a valid test IMO. Let me know if that works for you.